File: CS/MyComboBoxEdit/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using DevExpress.XtraEditors;



namespace MyComboBoxEdit
{
    public partial class FormMain : Form
    {
        private MyPopupHelper popupHelper;


        public FormMain()
        {
            InitializeComponent();
            popupHelper = new MyPopupHelper(panelControl1);   
        }
    }
}

File: VB/MyComboBoxEdit/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.XtraEditors



Namespace MyComboBoxEdit
	Partial Public Class FormMain
		Inherits Form
		Private popupHelper As MyPopupHelper


		Public Sub New()
			InitializeComponent()
			popupHelper = New MyPopupHelper(panelControl1)
		End Sub
	End Class
End Namespace
